src/main/java/org/zalando/nakadi/controller/SubscriptionStreamController.java

```diff
@@ -135,6 +143,13 @@ public StreamingResponseBody streamEvents(
             SubscriptionStreamer streamer = null;
             final SubscriptionOutputImpl output = new SubscriptionOutputImpl(response, outputStream);
             try {
+                if  (subscriptionDbRepository.getSubscription(subscriptionId).getEventTypes().stream()
+                        .map(etName -> floodService.isConsumptionBlocked(etName)).findFirst().orElse(false)) {
+                    response.setStatus(HttpStatus.TOO_MANY_REQUESTS.value());
```

Comment:
It look like if the floodService throws exceptions or fails internally, access will "fail closed" and the server will throw 500s; is that the intent?